modules/statistics/AnalyticsAdapter.js

```diff
@@ -5,6 +5,7 @@ NoopAnalytics.prototype.sendEvent = function () {};
 
 function AnalyticsAdapter() {
     this.browserName = RTCBrowserType.getBrowserName();
+    this.permanentProperties = {};
```

Comment:
I don't understand what permanent properties are. Technically, `this.permanentProperties` declares an equivalent of a Java class field. Then we're kind of supposed to document it with jsdocs anyway.